jumeg/connectivity/con_viz.py

```diff
@@ -561,22 +561,34 @@ def plot_grouped_connectivity_circle(yaml_fname, con, orig_labels,
     """
     import matplotlib.pyplot as plt
 
-    # read the yaml file with grouping
-    if op.isfile(yaml_fname):
-        with open(yaml_fname, 'r') as f:
-            label_groups = yaml.safe_load(f)
+    if isinstance(yaml_fname, str):
+        # read the yaml file with grouping
+        if op.isfile(yaml_fname):
+            with open(yaml_fname, 'r') as f:
+                label_groups = yaml.safe_load(f)
+        else:
+            print('%s - File not found.' % yaml_fname)
+            sys.exit()
+    elif isinstance(yaml_fname, list):
+        # should be a list of dictionaries
+        label_groups = yaml_fname
```

Comment:
I find it a bit unintuitive that yaml_fname could be a list. I guess that this is in preparation for getting rid of yamls but then we should rename the variable.

Also, now that I think about it, we should just make this a small function for reading the groups from the yaml instead of repeating the code twice in a row.